src/main/java/by/exadel/internship/entity/Internship.java

```diff
@@ -7,13 +7,19 @@
 import lombok.*;
 import org.hibernate.annotations.Fetch;
 import org.hibernate.annotations.FetchMode;
+import org.hibernate.annotations.GenericGenerator;
 import org.hibernate.annotations.Type;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.jpa.domain.AbstractAuditable;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
 import javax.persistence.*;
 import java.time.LocalDate;
 import java.util.List;
 import java.util.UUID;
 
+@EntityListeners({AuditingEntityListener.class})
```

Comment:
Is it nessessary? Class Auditable has it

Reply:
yaaa it's true. Removed it